apitools/base/py/credentials_lib.py

```diff
@@ -384,6 +387,21 @@ def _refresh(self, _):
     self.access_token = token
 
 
+def _GetRunFlowFlags():
+  if FLAGS is None:
```

Comment:
@craigcitro This differs from your approach below: `if hasattr(FLAGS, 'auth_local_webserver'):`

Is there a reason you didn't just check `FLAGS is not None`?